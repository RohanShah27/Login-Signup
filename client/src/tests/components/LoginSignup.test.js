import LoginSignUp from "../../components/Login-Signup/index";
import React from "react";
import { shallow } from "enzyme";

const wrapper = shallow(<LoginSignUp />);
describe("Test suite for Main home component", () => {
  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
  it("should check for on click events to mutate state", () => {
    expect(wrapper.find("#header").props().children).toBe("Log in");
    wrapper.find("#sign-up-click").simulate("click");
    expect(wrapper.find("#header").props().children).toBe("Sign up");
    wrapper.find("#log-in-click").simulate("click");
    expect(wrapper.find("#header").props().children).toBe("Log in");
    // Click on submit button
    wrapper.find("#submit-click").simulate("click");
  });
});
