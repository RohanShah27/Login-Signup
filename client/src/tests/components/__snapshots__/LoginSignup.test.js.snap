// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suite for Main home component should match snapshot 1`] = `ShallowWrapper {}`;
